src/DotTiled.Example/Program.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System.Reflection;
+using DotTiled.Serialization;
+
+namespace DotTiled.Example;
+
+class Program
```

Comment:
You can add to make the linter happy :)
```suggestion
public class Program
```